### PR TITLE
Problem: (Fix #56) No running azure 1-click deployment for croeseid testnet

### LINKED ISSUE #56: Problem: No running azure 1-click deployment for croeseid testnet
@crypto-matto 1-click deployment for the latest testnet is now available on the azure market place, 
Sample doc:  
https://thaler-testnet.crypto.com/docs/getting-started/thaler-testnet-azure-1click.html#azure-account-creation

### PR DESCRIPTION
**Pull request by crypto-matto**
Added Azure 1-click deployment page

### CODE REVIEW

File: docs/getting-started/testnet-azure-1click.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+## Step 2. Common Setup
+
+### Step 2-1. Search for Crypto.com Chain on Marketplace
+
+Sign in to your Microsoft Azure account and go to [Marketplace](https://portal.azure.com/#blade/Microsoft_Azure_Marketplace/MarketplaceOffersBlade/selectedMenuItemId/home). Search for "Crypto.com testnet-croeseid-1".
+
+![](./assets/azure_marketplace.png)
+
+### Step 2-2. Create Crypto.com Chain
```
Comment (lezzokafka) [inline docs/getting-started/testnet-azure-1click.md:19]:
```suggestion
### Step 2-2. Create a Virtual Machine
```

File: docs/getting-started/testnet-azure-1click.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+Choose the image and click "Create" to start creating the Crypto.com chain testnet node.
+
+#### 1. Basic
+
+![](./assets/azure_1click_basics.png)
+
+| Configuration           | Value                                                                                                            |
+| ----------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| _Region_                | DCsv2-series VMs for Azure are widely available in regions like _East US_, _Canada Central_, _UK South_, etc     |
```
Comment (lezzokafka) [inline docs/getting-started/testnet-azure-1click.md:29]: I think we dont have the regional restrictions now? @allthatjazzleo

**Comment by allthatjazzleo** [comment]
LGTM

File: docs/getting-started/testnet-azure-1click.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+-rwxr-xr-x  1 crypto crypto    7334 Aug 10 10:28 reconfig.sh*
+drwxr-xr-x  4 root   root      4096 Aug 10 10:28 release_binary/
+drwxr-xr-x  2 crypto crypto    4096 Aug 10 10:30 sockets/
+```
+
+
+
+## Step 3. Running a full node
+### Step 3-1. Use the reconfiguration script
+Now we use the reconfiguration script `reconfig.sh` to clean up the old data(if any) and obtain a fresh consensus public key, which is necessary for running a validator.
```
Comment (lezzokafka) [inline docs/getting-started/testnet-azure-1click.md:88]:
```suggestion
Now we use the reconfiguration script `reconfig.sh` to clean up the old data (if any) and obtain a fresh consensus public key, which is necessary for running a validator.
```

**Review by allthatjazzleo — APPROVED** [review]

File: docs/getting-started/testnet-azure-1click.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+drwxr-xr-x  2 crypto crypto    4096 Aug 10 10:35 .chain-abci/
+drwxr-xr-x  4 crypto crypto    4096 Aug 10 10:36 .tendermint/
+drwxr-xr-x  2 crypto crypto    4096 Aug 10 10:30 .tmkms/
+drwxr-xr-x  2 crypto crypto    4096 Aug 10 10:33 .tx-query-enclave/
+drwxr-xr-x  4 crypto crypto    4096 Aug 10 10:36 bin/
+-rwxr-xr-x  1 crypto crypto 2334994 Aug 10 10:28 convert_to_pubkey*
+drwx------  2 root   root     16384 Aug 10 10:20 lost+found/
+-rwxr-xr-x  1 crypto crypto    7334 Aug 10 10:28 reconfig.sh*
+drwxr-xr-x  4 root   root      4096 Aug 10 10:28 release_binary/
+drwxr-xr-x  2 crypto crypto    4096 Aug 10 10:30 sockets/
```
Comment (lezzokafka) [inline docs/getting-started/testnet-azure-1click.md:81]:
```suggestion
$ cd /chain/
$ ll
total 44
drwxr-xr-x  8 root   root   4096 Nov 23 14:11 ./
drwxr-xr-x 24 root   root   4096 Nov 27 03:30 ../
drwxr-xr-x  4 crypto crypto 4096 Nov 23 14:11 .chain-maind/
drwxr-xr-x  5 crypto crypto 4096 Nov 23 14:10 .tmkms/
drwxr-xr-x  2 crypto crypto 4096 Nov 23 14:12 bin/
-rwxr-xr-x  1 crypto crypto  316 Nov 23 14:03 cleanForAMI.sh*
drwxr-xr-x  2   3434   3434 4096 Jun  4  2019 node_exporter-0.18.1.linux-amd64/
-rwxr-xr-x  1 crypto crypto 5858 Nov 23 14:03 reconfig.sh*
drwxr-xr-x  2 root   root   4096 Nov 23 14:03 release_binary/
drwxr-xr-x  2 crypto crypto 4096 Nov 23 14:09 sockets/
```

File: docs/getting-started/testnet-azure-1click.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+$ journalctl -u chain-maind.service -f
+-- Logs begin at Mon 2020-11-23 13:21:03 UTC. --
+Nov 25 09:27:41 crypto-chain-test-node chain-maind[3209]: I[2020-11-25|09:27:41.854] Executed block                               module=state height=545512 validTxs=0 invalidTxs=0
+Nov 25 09:27:41 crypto-chain-test-node chain-maind[3209]: I[2020-11-25|09:27:41.866] Committed state                              module=state height=545512 txs=0 appHash=B989A1B3E225A6BCBB4D61072E83A9EB3F9C480CE8CFDBCC4B3469620D314F99
+Nov 25 09:27:48 crypto-chain-test-node chain-maind[3209]: I[2020-11-25|09:27:48.271] Executed block                               module=state height=545513 validTxs=1 invalidTxs=0
+Nov 25 09:27:48 crypto-chain-test-node chain-maind[3209]: I[2020-11-25|09:27:48.287] Committed state                              module=state height=545513 txs=1 appHash=D55A839445CD58DD8EAD27D8FA87ACAF88695BE47CDE4FFCD456421EB7800775
+...
+```
+
+Once the tendermint syncs to the latest block, the setup is done! You may move on to the next step if you want to join the network as a validator.
```
Comment (lezzokafka) [inline docs/getting-started/testnet-azure-1click.md:155]: The latest block height can be found by
`curl -s https://testnet-croeseid-1.crypto.com:26657/block | jq -r .result.block.header.height`

**Review by lezzokafka — COMMENTED** [review]
Looks alright👍 , just some minor suggestions.

**Review by lezzokafka — APPROVED** [review]
Cheers!

**Pull request merged**